src/main/kotlin/org/lsst/dax/albuquery/tasks/QueryTask.kt

```diff
@@ -63,8 +64,10 @@ class QueryTask(
 ) : Callable<QueryTask> {
 
     val columnAnalyzer: TableAndColumnExtractor
+    val phaseInfo: PhaseInfo
 
     init {
+        phaseInfo = PhaseInfo(phase="READY")
```

Comment:
Isn't this supposed to be PENDING?

2.1.3. Execution Phase - http://www.ivoa.net/documents/UWS/20161024/REC-UWS-1.1-20161024.pdf

Also, we might want to make some kind of enum for this, because it's a state, and has only a certain set of values.  Perfect use for an enum.  We can always convert it back to string somewhere.  That will prevent the state from typos and invalid settings, etc.